.eslintrc.json

```diff
@@ -1,17 +1,169 @@
 {
-    "extends": "./.eslintrc-platform.json",
```

Comment:
The purpose is to use same base config with mattermost-webapp. You can write only additional rules here. What was actually needed for you?